modules/persistent-storage-local-removing-devices.adoc

```diff
@@ -24,33 +19,84 @@ Deleting a persistent volume that is still in use can result in data loss or cor
 +
+[source, terminal]
+----
+$ oc get pv --selector storage.openshift.com/owner-name=<LV_LVS_name> <1>
+----
+<1> `<LV_LVS_name>` is the name of the LV or LVS.
++
+.Example output
 [source,terminal]
 ----
-$ oc edit localvolume <name> -n openshift-local-storage
+NAME                CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM                 STORAGECLASS   VOLUMEATTRIBUTESCLASS   REASON   AGE
+local-pv-3fa1c73    5Gi        RWO            Delete           Available                         slow           <unset>                          28s
+local-pv-1cec77cf   30Gi       RWX            Retain           Bound       openshift/storage     my-sc          <unset>                          168d
 ----
++
+Bound PVs have a status of `Bound` and their corresponding PVCs appear in the `CLAIM` column. In the preceding example, PV `local-pv-1cec77cf` is bound, and its PVC is `openshift/storage`.
 
-.. Navigate to the lines under `devicePaths`, and delete any representing unwanted disks.
+.. Delete corresponding PVCs of bound PVs owned by the LV or LVS being deleted by running the following command:
++
+[source, terminal]
+----
+$ oc delete pvc <name>
+----
++ 
+In this example, you would delete PVC `openshift/storage`.
 
-. Delete any persistent volumes created.
+. Delete the LVs or LVSs by running the applicable following command:
++
+.Command for deleting LV
 +
 [source,terminal]
 ----
-$ oc delete pv <pv-name>
+$ oc delete lv <name>
+----
++
+or
++
+.Command for deleting LVS
+[source,terminal]
+----
+$ oc delete lvs <name>
 ----
 
-. Delete directory and included symlinks on the node.
+. If any PV owned by the LV or LVS has a `Retain` reclaim policy, back up any important data, and then delete the PV:
 +
-[WARNING]
+[NOTE]
 ====
-The following step involves accessing a node as the root user. Modifying the state of the node beyond the steps in this procedure could result in cluster instability.
+PVs with a `Delete` policy are automatically deleted when you delete the LVs or LVS.
 ====
 +
+.. To find PVs with `Retain` reclaim policy, run the following command:
++
+[source,terminal]
+----
+$ oc get pv
+----
++
+.Example output
 [source,terminal]
 ----
-$ oc debug node/<node-name> -- chroot /host rm -rf /mnt/local-storage/<sc-name> <1>
+NAME                CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM                STORAGECLASS   REASON   AGE
+local-pv-1cec77cf   30Gi       RWX            Retain           Available                        my-sc                   168d
 ----
-<1> The name of the storage class used to create the local volumes.
++
+In this example,  PV `local-pv-1cec77cf` has a `Retain` reclaim policy and needs to be manually deleted.
+
+.. Back up any important data on this volume.
+
+.. Delete the PV by running the following command:
++
+[source,terminal]
+----
+$ oc delete pv <name>
+----
++
+In this example, delete PV local-pv-3fa1c73`.
```

Comment:
don't we want to delete local-pv-1cec77cf in that case?
@dobsonj

Reply:
Yes! Missed that. It's fixed now.